src/main/java/dinkplugin/message/DiscordMessageHandler.java

```diff
@@ -79,6 +79,9 @@ public void createMessage(String webhookUrl, boolean sendImage, @NonNull Notific
         if (mBody.getPlayerName() == null)
             mBody = mBody.withPlayerName(Utils.getPlayerName(client));
 
+        if (mBody.getPlayerUrl() == null)
+            mBody = mBody.withPlayerUrl(config.playerLookupService().playerUrl(mBody.getPlayerName()));
```

Comment:
Instead of storing the player url in `NotificationBody`, let's just pass `PlayerLookupService` (pulled from config) to `injectContent`